src/lib/components/Swap/Summary.fixture.tsx

```diff
@@ -41,7 +41,7 @@ function Fixture() {
   return trade ? (
     <Modal color="dialog">
       <SummaryDialog
-        onConfirm={() => void 0}
+        onConfirm={async () => void 0}
```

Comment:
what's happening here? (I've never used the void operator, and the docs I just read aren't helping me understand this usage)